packages/wonder-blocks-tooltip/components/tooltip-tail.js

```diff
@@ -9,6 +9,15 @@ import Spacing from "@khanacademy/wonder-blocks-spacing";
 import type {getRefFn, Placement, Offset} from "../util/types.js";
 
 export type Props = {|
+    /**
+     * Whether we should use the default white background color or switch to a
+     * different bg color.
+     *
+     * NOTE: Added to support custom popovers
+     * @ignore
+     */
+    color: $Keys<typeof Colors>,
```

Comment:
Used this here to only allow valid WB colors.

Reply:
Nice!